### PR TITLE
[WIP] Add procedures for setting up events

### PR DESCRIPTION
**Pull request by loriking**

### CODE REVIEW

File: proceedures.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+### 3. Create a card in Trello
+Procedures:
+
+Copy [this card](https://trello.com/c/6xVLVl5A) to set up a new workshop:
+
++ Adjust title to the correct date. Example: "Event 2018.03.08"
++ Set the due date on the card matching the event date
++ Make sure the "Event" blue tag is present
++ Remove the extra logos. Make the correct one the card cover.
++ Define the door responsible and the MC. Assign them as members of the card so they get notification of any changes.
```
Comment (jmbriano) [inline proceedures.md]: "Define the door responsible and the MC. Assign them as members of the card so they get notification of any changes." -> I would rephrase to: "Assign the card to you so the rest of the organisers can easily see you are the owner. Once defined, fill in the specific roles in the card description (MC, Door Person, and Pair responsible)"

File: proceedures.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
++ Remove everything up to the line below and fill in the rest.
```
Comment (jmbriano) [inline proceedures.md]: This line should be removed. It made sense in the trello card, but not anymore.

**Comment by jmbriano** [comment]
@loriking as discussed in the last Monthly, I completed this process. Also restructured a bit the folders. 

how does it look?

**Comment by daverick** [comment]
Lgtm

**Comment by amaliacardenas** [comment]
LGTM as well. 👍

**Comment by jmbriano** [comment]
@loriking review and merge?
